From febbb5f7e75dda4a60513ec39220f1599008a2c3 Mon Sep 17 00:00:00 2001
From: Package Bot <bot@chainguard.dev>
Date: Tue, 1 Sep 2026 06:15:39 +0000
Subject: [PATCH] fix: add missing source files to Makefile.am for v2.16

The Makefile.am was not updated to include new source files added
in v2.16. The meson.build has the correct list. Add the missing:
- gpu-tab.cpp / gpu-tab.h
- cpu/frequency.cpp / cpu/frequency.h
- cpu/xe_gpu.cpp
- devices/device_manager.cpp
- devices/xe-gpu.cpp / devices/xe-gpu.h
- measurement/measurement_manager.cpp
- report/report-formatter-md.cpp / report/report-formatter-md.h
---
 src/Makefile.am | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/src/Makefile.am b/src/Makefile.am
index 5734f7b..8388e60 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -13,12 +13,16 @@ powertop_SOURCES = \
 	display.h \
 	lib.cpp \
 	lib.h \
+	gpu-tab.cpp \
+	gpu-tab.h \
 	main.cpp \
 	powertop.css \
 	\
 	calibrate/calibrate.cpp \
 	calibrate/calibrate.h \
 	cpu/abstract_cpu.cpp \
+	cpu/frequency.cpp \
+	cpu/frequency.h \
 	cpu/cpu.cpp \
 	cpu/cpu.h \
 	cpu/cpu_core.cpp \
@@ -33,6 +37,7 @@ powertop_SOURCES = \
 	cpu/intel_cpus.cpp \
 	cpu/intel_cpus.h \
 	cpu/intel_gpu.cpp \
+	cpu/xe_gpu.cpp \
 	cpu/rapl/rapl_interface.cpp \
 	cpu/rapl/rapl_interface.h \
 	devices/ahci.cpp \
@@ -45,10 +50,13 @@ powertop_SOURCES = \
 	devices/devfreq.h \
 	devices/device.cpp \
 	devices/device.h \
+	devices/device_manager.cpp \
 	devices/gpu_rapl_device.cpp \
 	devices/gpu_rapl_device.h \
 	devices/i915-gpu.cpp \
 	devices/i915-gpu.h \
+	devices/xe-gpu.cpp \
+	devices/xe-gpu.h \
 	devices/network.cpp \
 	devices/network.h \
 	devices/rfkill.cpp \
@@ -67,6 +75,7 @@ powertop_SOURCES = \
 	measurement/extech.h \
 	measurement/measurement.cpp \
 	measurement/measurement.h \
+	measurement/measurement_manager.cpp \
 	measurement/sysfs.cpp \
 	measurement/sysfs.h \
 	measurement/opal-sensors.cpp \
@@ -99,6 +108,8 @@ powertop_SOURCES = \
 	report/report-formatter-base.h \
 	report/report-formatter-csv.cpp \
 	report/report-formatter-csv.h \
+	report/report-formatter-md.cpp \
+	report/report-formatter-md.h \
 	report/report-formatter-html.cpp \
 	report/report-formatter-html.h \
 	report/report-formatter.h \
-- 
2.55.0
